pkg/midstream/write.go

```diff
@@ -102,20 +93,18 @@ func (m *Midstream) mergeKustomization(existing *kustomizetypes.Kustomization) {
 	newResources := findNewStrings(m.Kustomization.Resources, existing.Resources)
 	m.Kustomization.Resources = append(existing.Resources, newResources...)
 
-	m.Kustomization.CommonAnnotations = mergeMaps(m.Kustomization.CommonAnnotations, existing.CommonAnnotations)
```

Comment:
this is needed

Reply:
hmmm, before gitops tracers were added, this statement never existed. why is it needed now?